src/cli/climode.go

```diff
@@ -17,7 +18,7 @@ func printResults(results []result.Result) {
 	for _, r := range results {
 		fmt.Printf("%v (%.2f) -----\n\t\"%v\"\n\t\"%v\"\n\t\"%v\"\n\t-", r.Rank, r.Score, r.Title, r.URL, r.Description)
 		for seInd := uint8(0); seInd < r.TimesReturned; seInd++ {
-			fmt.Printf("%v", r.EngineRanks[seInd].SearchEngine)
+			fmt.Printf("%v", r.EngineRanks[seInd].SearchEngine.ToLower())
```

Comment:
we should have a method that returns the correctly written name (ex. DuckDuckGo, not duckduckgo)